packages/govuk-frontend/src/govuk/components/phase-banner/_index.scss

```
@@ -17,7 +17,16 @@
   }
 
   .govuk-phase-banner__content__tag {
+    @include govuk-font($size: 16);
     margin-right: govuk-spacing(2);
+
+    // When forced colour mode is active, for example to provide high contrast,
+    // the background colour of the tag is the same as the rest of the page. To ensure
+    // that that tag is perceived as separate the rest of the text in the phase banner,
```

Got a double "that that" and a missing word in this block

If you can fix this one and use the same comment further down too?

```suggestion
    // that tag is perceived as separate to the rest of the text in the phase banner,
```